graylog2-server/src/main/java/org/graylog/plugins/views/search/errors/SearchTypeError.java

```diff
@@ -18,27 +18,54 @@
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import org.graylog.plugins.views.search.Query;
-import org.graylog.plugins.views.search.Query;
 
 import javax.annotation.Nonnull;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
```

Comment:
Note to the reviewer(s): I was torn between adding the special case to the existing `SearchTypeError` and adding a new kind of error. i went with adding it in, because I did not want to complicate `ElasticsearchBackend` with detecting the special case and constructing an instance of an alternative error class.